lighty-core/lighty-controller/src/test/java/io/lighty/core/controller/util/FileToDatastoreUtilsTest.java

```diff
@@ -77,6 +90,12 @@ public void tearDown() throws Exception {
 
     @Test
     public void testTopLevelNode() throws Exception {
+        // Import inner-case and test choice node
+        importFile(CASE_CONTAINER_PATH, INNER_CASE_YIID, ImportFileFormat.JSON);
+        ChoiceContainer choiceContainer = readDataFromDatastore(
+                CHOICE_CONTAINER_YII);
```

Comment:
Please edit this, so it will be in one line:
ChoiceContainer choiceContainer = readDataFromDatastore(CHOICE_CONTAINER_YII);

Reply:
Fixed